--- google-api-php-client-1.1.2/autoload.php	2015-01-23 00:06:59.492462347 +0200
+++ google-api-php-client-1.1.2/autoload.php	2015-01-23 10:59:15.270389353 +0200
@@ -27,7 +27,11 @@
-  $filePath = dirname(__FILE__) . '/src/' . implode('/', $classPath) . '.php';
+  array_shift($classPath);
+  $filePath = dirname(__FILE__) . '/' . implode('/', $classPath) . '.php';
   if (file_exists($filePath)) {
     require_once($filePath);
   }
 }
 
-spl_autoload_register('google_api_php_client_autoload');
+// avoid registering twice under composer
+if (!class_exists('Composer\Autoload\ClassLoader', false)) {
+	spl_autoload_register('google_api_php_client_autoload');
+}
