--- php-google-api-0.6.1/src/auth/Google_P12Signer.php~	2013-02-28 06:07:44.000000000 +0200
+++ php-google-api-0.6.1/src/auth/Google_P12Signer.php	2013-07-03 10:20:01.936330882 +0300
@@ -58,10 +58,6 @@
   }
 
   function sign($data) {
-    if(version_compare(PHP_VERSION, '5.3.0') < 0) {
-      throw new Google_AuthException(
-        "PHP 5.3.0 or higher is required to use service accounts.");
-    }
     if (!openssl_sign($data, $signature, $this->privateKey, "sha256")) {
       throw new Google_AuthException("Unable to sign data");
     }
