--- google-api-php-client-1.1.2/src/Google/Signer/P12.php~	2014-11-25 16:35:03.000000000 +0200
+++ google-api-php-client-1.1.2/src/Google/Signer/P12.php	2015-01-22 23:49:13.673899323 +0200
@@ -76,11 +76,6 @@
 
   public function sign($data)
   {
-    if (version_compare(PHP_VERSION, '5.3.0') < 0) {
-      throw new Google_Auth_Exception(
-          "PHP 5.3.0 or higher is required to use service accounts."
-      );
-    }
     $hash = defined("OPENSSL_ALGO_SHA256") ? OPENSSL_ALGO_SHA256 : "sha256";
     if (!openssl_sign($data, $signature, $this->privateKey, $hash)) {
       throw new Google_Auth_Exception("Unable to sign data");
